.github/workflows/linux-dispatch-build-bottle.yml

```diff
@@ -24,8 +24,6 @@ jobs:
     container:
       image: ghcr.io/homebrew/ubuntu16.04:master
       options: --user=linuxbrew
-    env:
-      HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com/
```

Comment:
Was this set intentionally in order to avoid Linuxbrew?

Reply:
It'll be so we don't upload to Linuxbrew on Bintray. We're not actually building any bottles here yet, though. CC @dawidd6 who added this and @iMichka who has been doing Linux CI: https://github.com/Homebrew/homebrew-core/commit/42ec8917edd6edee6bfd0f331dbcd0ea6acf66d5

@Bo98 Do you consider this blocking?

Reply:
> It'll be so we don't upload to Linuxbrew on Bintray.

It’ll also avoid using existing Linuxbrew bottles for dependencies.

> Do you consider this blocking?

Since we’re not using this workflow then no, but it’s worth remembering to fix this workflow as there’s now no upload step.

Reply:
I think we uploaded the patchelf and binutils bottles. That's all. Maybe the hello bottle for a test, once. But right now there is not Linux bottle uploading  in this repo.